priv/templates/nimble_template/.github/workflows/README.md

```diff
@@ -0,0 +1,61 @@
+# Github Actions
+
+The following workflows are supported. Depends on what you choose when generating the project, some or all of them would be available:
+
+- Test
+- [Deploy to Heroku](#deploy-to-heroku-workflow)
+
+Please refer to the following sections for usage instruction.
+
+## Deploy to Heroku Workflow
```

Comment:
Should we only generate this content when the user chooses `Deploy to Heroku` option, because if the user doesn't choose it, it doesn't make sense to have this section in the README file

Reply:
<img width="830" alt="Screen Shot 2021-12-24 at 13 32 43" src="https://user-images.githubusercontent.com/1896814/147325736-7741fd65-49b6-4cc3-b421-efc2fb3dc0fe.png">

do you have any ideas to prepend the content to 2 places when a new workflow option is selected?

e.g. if the user selects to generate the Wiki workflow, we also need to append to the above 2 places.
As a workaround, I added a sentence to the README file

> The following workflows are supported. Depends on what you choose when generating the project, some or all of them would be available:

Reply:
We can put the condition into this README.md file @longnd I will try to adjust that on a separate PR